bayesian_torch/layers/flipout_layers/conv_flipout.py

```diff
@@ -173,16 +176,18 @@ def forward(self, x, return_kl=True):
 
         delta_kernel = (sigma_weight * eps_kernel)
 
-        kl = self.kl_div(self.mu_kernel, sigma_weight, self.prior_weight_mu,
-                         self.prior_weight_sigma)
+        if return_kl:
```

Comment:
Hey, I think only computing the kl value if return_kl is set to True might break things if we are plugging bayesian layers on a deterministic model. 

Maybe the best way could be if `self.dnn_to_bnn_flag` is set to True, we set it to null, but not computing it at all might just break everything - and we don't have unit tests yet to ensure things won't break.

Reply:
Hey @piEsposito , Do you have any specific scenarios in mind that this would break? With these changes, kl can be computed in kl_loss() function (https://github.com/IntelLabs/bayesian-torch/pull/11/files#diff-a588ca678816135810210bacf4645009ff6ecd3f16626713fd576db4aa603b07R126). For example, see https://github.com/IntelLabs/bayesian-torch/pull/11/files#diff-ade00648d59ef89627b50d68cb88f47ffc6d4832dd4c366902285fce80fec709R157 .